src/tests/architect_tests/test_feature_group_creator.py

```diff
@@ -3,94 +3,89 @@
 
 
 def test_table_group():
-    group = FeatureGroupCreator(
-        definition={'tables': ['one', 'three']}
-    )
+    group = FeatureGroupCreator(definition={"tables": ["one", "three"]})
 
-    subsets = group.subsets({
-        'one': ['col_a', 'col_b', 'col_c'],
-        'two': ['col_a', 'col_b', 'col_c'],
-        'three': ['col_a', 'col_b', 'col_c'],
-    })
+    subsets = group.subsets(
+        {
+            "one": ["col_a", "col_b", "col_c"],
+            "two": ["col_a", "col_b", "col_c"],
+            "three": ["col_a", "col_b", "col_c"],
+        }
+    )
```

Comment:
This may be the only reformatting I don't like; it's actually less compact. But I can understand where the unchanged behavior can cause awkwardness, so I can accept this.